en/application.texy

```diff
@@ -163,28 +163,28 @@ Simply write the familiar `Presenter:action` pair instead of the real URL and in
 <a href="https://example.com/product/456">product detail</a>
 ```
 
-The previously mentioned router is in charge of generating the URL. In fact, routers in Nette are unique in that they can perform not only transformations from a URL to a pair of presenter:action, but also vice versa, ie generate a URL from the name of the presenter + action + parameters.
-Thanks to this, in Nette you can completely change the form of the URL in the whole finished application without changing a single character in the template or presenter. Just by modifying the router. Many programmers find this amazing.
+The previously mentioned router is in charge of generating the URL. In fact, routers in Nette are unique in that they can perform not only transformations from a URL to a pair of presenter:action, but also vice versa  generate a URL from the name of the presenter + action + parameters.
+Thanks to this, in Nette you can completely change the form of the URL in the whole finished application without changing a single character in the template or presenter just by modifying the router. Many programmers find this amazing.
 
 
 
 Components
 ==========
 
-We have one more thing to tell you about presenters: they have a built-in component system. Witnesses may know something similar from Delphi or ASP.NET Web Forms, and React or Vue.js is built on something remotely similar. In the world of PHP frameworks, this is a completely unique feature.
+We have one more thing to tell you about presenters: they have a built-in component system. You may remember something similar from Delphi or ASP.NET Web Forms. React or Vue.js is built on something remotely similar. In the world of PHP frameworks, this is a completely unique feature.
```

Comment:
Tak mělo být "pamětníci", totiž nepřepokládám, že nějaký čtenář by si to pamatoval, ale není od věci dostat do docky i klíčová slova jako React a Vue :)

Reply:
Ja si prave nejsem jisty ze anglictina ma podobne slovo. Witness je spis svedek :) popremyslim jak to napsat jinak

Reply:
V dokumentu o Epsteinovi používali slovo „survivor“ :-)

Reply:
Nemohl by tam byt treba neco jako "Older of you may remember something similar from Delphi or ASP.NET Web Forms" ?